apps/openchallenges/thumbor/README.md

```diff
@@ -4,17 +4,29 @@
 
 ### Prepare the service
 
+Run this command to prepare this project, including creating the config file `.env`.
+
 ```console
 nx prepare openchallenges-thumbor
```

Comment:
Nit: I'd also add a note about downloading the Docker image first (I forgot to do that and was met with a `access denied` error 😅 )

Reply:
Nice catch! I added a section below about building the Docker image.